src/components/Accordion/AccordionSummary/AccordionSummary.scss

```diff
@@ -0,0 +1,40 @@
+@use '../../variables';
+@use '../../../../styles/mixins';
+
+$block: '.#{variables.$ns}accordion-summary';
+
+@mixin accordion-item-trigger($size) {
+    #{$block}__trigger {
+        @if $size == 's' or $size == 'm' {
+            @include mixins.text-subheader-1();
+        }
+        @if $size == 'l' or $size == 'xl' {
+            @include mixins.text-subheader-2();
+        }
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
```

Comment:
This is the common styles for trigger, i guess it would be better to set them separately from size modifiers to single rule